README.md

```diff
@@ -185,7 +186,6 @@ group.background {
 group.utility {
     // Run on utility queue, in parallel to the previous block
 }
-group.wait()
```

Comment:
I think I remove it because the next few lines document show the wait alternative and the non blocking alternative for finish.So I remove it to let the users choose which alternative would be better in their use case.
But if it's confused, I think we should add it back.